src/datumaro/plugins/data_formats/coco/base.py

```diff
@@ -450,7 +449,8 @@ def _parse_field(
     ) -> Any:
         value = ann.get(key, NOTSET)
         if value is NOTSET:
-            raise MissingFieldError(key)
+            log.warning(f"field '{key}' is not existed in the annotation file.")
+            return None
```

Comment:
I think that just removing this error raise is not appropriate. It can be a problem for those mandatory fields

https://github.com/openvinotoolkit/datumaro/blob/183a69e1ac8310ba9d8c4fec5f22067328fa1fa6/src/datumaro/plugins/data_formats/coco/base.py#L465
https://github.com/openvinotoolkit/datumaro/blob/183a69e1ac8310ba9d8c4fec5f22067328fa1fa6/src/datumaro/plugins/data_formats/coco/base.py#L480

They are not only mandatory, but also have no handling logic if `None` is given. This can introduce a bug which is not easily understandable if the malformed JSON input is given.

Reply:
Instead, I have updated by making only "segmentation" optional. Could you review again?